Test/Core/Components/RigidBodyTest.cs

```diff
@@ -1,21 +1,54 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.IO;
-using System.Reflection;
-
-using Duality;
-using Duality.Cloning;
+﻿using System.Linq;
+using System.Threading;
 using Duality.Components;
 using Duality.Components.Physics;
-
+using Duality.Resources;
 using NUnit.Framework;
 
 namespace Duality.Tests.Components
 {
 	[TestFixture]
 	public class RigidBodyTest
 	{
+		[Test] public void FallingBallOnPlatform()
+		{
+			var scene = new Scene();
```

Comment:
Don't use `var` if it can be reasonably avoided, keep variable definitions explicit instead.